README.rst

```diff
@@ -60,7 +60,18 @@ See the output of ``deepspeech -h`` for more information on the use of ``deepspe
 
 **Table of Contents**
 
-
+* `Trying out DeepSpeech with examples <examples/EXAMPLES.rst>`_
```

Comment:
Looks good, I'm just wondering if it should be at the top. @reuben @kdavis-mozilla  ?

Reply:
Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

Reply:
I would argue that it is the first item on the list.
If you see a project for the first time, I think a good start is to play around with the examples.


> Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

Looks also good I think.

Reply:
>  If you see a project for the first time, I think a good start is to play around with the examples.

The problem with this assertion is that it holds depending on context, so we can't satisfy everybody :/

Reply:
> Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

@bprfh would work for me